composer.json

```
@@ -50,31 +50,29 @@
         "fakerphp/faker": "^1.21",
         "google/protobuf": "^3.21",
         "larastan/larastan": "^2.6.1",
-        "laravel/framework": "^9 || ^10",
+        "laravel/framework": "^11",
```

```suggestion
        "laravel/framework": "^9 || ^10 || ^11",
```

Just for testing to force the update.